gg-pac-team3-front/src/components/layout/Nav.jsx

```diff
@@ -0,0 +1,33 @@
+import logo from '../../assets/logos/blue.png';
+import styles from './nav.module.css';
+import { NavLink, Outlet} from "react-router-dom";
+
+const Nav = (props) => {
+
+    const {isLoggedIn} = props;
+
+    const navLinks = isLoggedIn? ['Dashboard', 'Account', 'Logout'] : ['About', 'Login', 'Register'];
+
+    return(
+         <>
+            <div className={styles.nav}>
+                <img src={logo}/>
+                <div className={styles.list}>
+                    <ul>
+                        { navLinks.map((link) => (
+                            <li key={link}>
+                                <NavLink to={link.toLowerCase()}>{link}</NavLink>
+                            </li>
+                        ))
+
+                        }
+                    </ul>
+                </div>
+                </div>
+                <div>
+                    <Outlet />
```

Comment:
I don't think this will work. In this example (https://reactrouter.com/en/main/components/outlet), the outlet is placed in a component that is a parent route to other components/pages. Thus, when we go to one of this child paths, it shows the `Dashboard` component and fits in the component assigned to the specific child route in the place of the `Outlet`.

Here, I think we should have a route structure similar to what the example has, like:

```
function App() {
  ....
  return (
    <Routes>
      <Route path="/" element={<Panel/>}>
        <Route
          path="frontPage"
          element={<FrontPage/>}
        />
        // Other routes in the future go here
      </Route>
    </Routes>
  );
}
```

Then our panel page should be like:
```
return (
    <Nav/>
    <Outlet/>
);
```

and the front page can stay as it currently is. This way, `Panel` just defines the Navbar -> Page body structure and whatever page the user navigates to can be slotted into the Outlet.

Reply:
@lauradurandv did you figure this out?

Reply:
I believe I figured it out.